content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,185 @@
+
+* Understand Istio [authentication policy](/docs/concepts/security/#authentication-policies) and related
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) concepts.
+
+* Have a Kubernetes cluster with Istio installed, with mutual TLS enabled (e.g use `install/kubernetes/istio-demo-auth.yaml` as described in
+[installation steps](/docs/setup/kubernetes/quick-start/#installation-steps)).
+
+### Setup
+
+This tutorial runs in a new namespace called `rbac-groups-test-ns`, with two services, `httpbin` and `sleep`,
+both running with an Envoy sidecar proxy. The following commands set an environmental variable to store the
+name of the namespace, creates the namespace, and starts the two services. Before running the following commands,
+you need to enter the directory containing the Istio installation files.
+
+{{< text bash >}}
+$ export NS=rbac-groups-test-ns
+$ kubectl create ns $NS
+$ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n $NS
+$ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n $NS
+{{< /text >}}
+
+To verify that `httpbin` and `sleep` services are running and `sleep` is able to reach `httpbin`, the
+following curl command should succeed with HTTP code 200.
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+{{< /text >}}
+
+In addition, verify that no authentication policies have been defined in the newly created namespace.
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io -n $NS
+No resources found.
+{{< /text >}}
+
+## Configure authentication and authorization
+
+### Configure JWT authentication with mutual TLS
+
+Apply an authentication policy to require both mutual TLS and JWT authentication for `httpbin`:
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n $NS -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "require-mtls-jwt"
+spec:
+  targets:
+  - name: httpbin
+  peers:
+  - mtls: {}
+  origins:
+  - jwt:
+      issuer: "testing@secure.istio.io"
+      jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+  principalBinding: USE_ORIGIN
+EOF
+{{< /text >}}
+
+With the above authentication policy, a valid JWT is needed to access the httpbin service. The JWT must
+be signed by the JWKS endpoint defined in the above policy. In this tutorial,
+the [JWKS endpoint]({{< github_file >}}/security/tools/jwt/samples/jwks.json) is from the Istio code
+base and the JWT is from a [sample JWT]({{< github_file >}}/security/tools/jwt/samples/groups-scope.jwt), which
+contains a JWT claim with the claim key being `groups` and the claim value being a list of strings [`"group1"`, `"group2"`].
```

Comment:
It might be worth to mention that the JWT claim could either be a simple `string` or a `list of strings`, both type are supported.

Reply:
Done.